_posts/2019-05-10-continuous-deployment-the-first-step-on-the-road-to-recovery.markdown

```diff
@@ -0,0 +1,63 @@
+# Continuous Deployment - the first step on your road to recovery
+
+** HMM, does this still fit?**
+[Systems thinking]() in DevOps is about improving the flow of work into production.  Hand-offs, re-work and batching are all enemies of flow.  Teams don't intentionally look to reduce flow.  Hand-offs and approvals are mechanisms introduced in an attempt to tackle the risks of delivery.  Value is generated through change.  Change is the enemy of stability.  XXX said "Systems architecture is a reflection of organisational structure".  ***I say the deployment pipeline is a reflection of the reporting structure.***.  Continuous deployment is a goal for most teams; a lofty ambition that other people achieve.  For Mechanical Rock, continuous deployment is our starting point.  And continuous deployment starts with continuous delivery and a suitable branching strategy.
+
+Your branching strategy is a fundamental decision for development teams and has profound effects upon your working practices.  Making a poor choice has knock on effects that can limit improvements in other areas.  
+
+The goal of [continuous delivery](https://continuousdelivery.com/) is:
+> To make deployments—whether of a large-scale distributed system, a complex production environment, an embedded system, or an app—predictable, routine affairs that can be performed on demand.
+> We achieve all this by ensuring our code is always in a deployable state
+
+Or more simply: “if it hurts, do it more often, and bring the pain forward.”
+
+# Damming the Flow
+
+A common branching strategy adopted by many teams is git flow.  
+
+[!GitFlow diagram](../img/continuous-deployment-the-first-step)*
```

Comment:
- If you just want to render an image, the `!` goes outside the `[]`. 😄 
- Does the trailing `*` do anything?

Reply:
Link to footnote. :+1